docs/Shell-Completion.md

```diff
@@ -11,8 +11,8 @@ You must configure your shell to enable its completion support. This is because
 To make Homebrew's completions available in `bash`, you must source the definitions as part of your shell's startup. Add the following to your `~/.bash_profile` file:
 
 ```sh
-HOMEBREW_PREFIX=$(brew --prefix)
 if type brew &>/dev/null; then
+  HOMEBREW_PREFIX=$(brew --prefix)
```

Comment:
Add quotes to this.

Reply:
@petdance any news on this?

Reply:
I gave it a thumbs up.  Or are you asking me to redo the PR with the quotes?

Reply:
> are you asking me to redo the PR with the quotes?

Please do. I thought your thought your 👍 implied that you would be adding the quotes.